Fix duplicate label warning

demonicblue opened the pull request:
Replace include directives with toctree. Otherwise the *yocto-project-integration* label is seen twice and generates a warning.

Signed-off-by: Henrik Hugo <hhugo@luxoft.com>

demonicblue commented:
This PR requires some discussion. Should the same change be applied to all files that are included by the 'include' directive?
Does this impair the ability to use files in a toctree?

tobsan reviewed (changes requested):
I think we have discussed earlier on ways to get around this by using toctree instead of include for most of these things. Could you try that instead?

demonicblue commented:
Yes I can but it will change the structuring of the page somewhat.

Commit:
Fix duplicate label warning

Replace include directives with toctree in order to avoid
warning from sphinx. Otherwise, the 'yocto-project-integration'
label is seen twice and generates a warning.

Signed-off-by: Henrik Hugo <hhugo@luxoft.com>
